### ISSUE TITLE
使用Deeplabv3P和gcnet训练mIou仅有35.9%

### ISSUE THREAD

**Issue by AlwaysGemini** [PaddlePaddle/PaddleSeg#940]
GPU：RTX 3070
Paddleseg 2.0.0
CUDA：11.0
修改根目录下的config文件训练自己的数据集，其他模型训练完mIou都能到97%、98%左右，只有Deeplabv3P和gcnet的mIou仅有35.9%，而且十分稳定，几乎就是一条平的线

如右图：https://imgtu.com/i/cMQr6g
底下的Deeplabv3P和gcnet几乎重合了

deeplabv3p.yml：
`_base_: '../_base_/my_base.yml'
batch_size: 1

loss:
  types:
    - type: CrossEntropyLoss
  coef: [1]

model:
  type: DeepLabV3P
  backbone:
    type: ResNet101_vd
    output_stride: 8
    multi_grid: [1, 2, 4]
    pretrained: https://bj.bcebos.com/paddleseg/dygraph/resnet101_vd_ssld.tar.gz
  num_classes: 2
  backbone_indices: [0, 3]
  aspp_ratios: [1, 12, 24, 36]
  aspp_out_channels: 256
  align_corners: False
  pretrained: null
`

gcnet.yml：
`_base_: '../_base_/my_base.yml'

batch_size: 1

loss:
  types:
    - type: CrossEntropyLoss
  coef: [1, 0.4]

model:
  type: GCNet
  backbone:
    type: ResNet101_vd
    output_stride: 8
    pretrained: https://bj.bcebos.com/paddleseg/dygraph/resnet101_vd_ssld.tar.gz
  gc_channels: 512
  ratio: 0.25
  enable_auxiliary_loss: True
  align_corners: False
  pretrained: null
  num_classes: 2`

其中my_base.yml是我所有模型的配置文件共用的
`iters: 30000

train_dataset:
  type: Dataset
  dataset_root: G:\Dataset\paddle
  train_path: G:\Dataset\paddle\train.txt
  num_classes: 2
  transforms:
    - type: RandomHorizontalFlip
    - type: RandomVerticalFlip
    - type: RandomPaddingCrop
      crop_size: [512, 512]
      im_padding_value: [127.5, 127.5, 127.5]
      label_padding_value: 255
    - type: RandomRotation
      max_rotation: 180
      im_padding_value: [127.5, 127.5, 127.5]
      label_padding_value: 255
    - type: RandomBlur
    - type: RandomDistort
      brightness_range: 0.4
      contrast_range: 0.4
      saturation_range: 0.4
    - type: Resize
      target_size: [512, 512]
    - type: Normalize
  mode: train

val_dataset:
  type: Dataset
  dataset_root: G:\Dataset\paddle
  val_path: G:\Dataset\paddle\val.txt
  num_classes: 2
  transforms:
    - type: Resize
      target_size: [512, 512]
    - type: Normalize
  mode: val

optimizer:
  type: sgd
  momentum: 0.9
  weight_decay: 4.0e-5

learning_rate:
  value: 0.01
  decay:
    type: poly
    power: 0.9
    end_lr: 0.00001

model:
  num_classes: 2
`

**Comment by michaelowenliu** [comment]
@michaelowenliu Hi, could you please try ResNet50_vd as the backbone rather than 101?

**Comment by AlwaysGemini** [comment]
> @michaelowenliu Hi, could you please try ResNet50_vd as the backbone rather than 101?

I just tried to use Resnet50_VD as backbone, and the result of mIou has been around 29.53%.

**Comment by michaelowenliu** [comment]
@AlwaysGemini Hi, that's weird. How about turning off data augmentation for colour distortion?
